build.gradle

```diff
@@ -4,7 +4,7 @@ subprojects {
     apply plugin: 'idea'
     apply plugin: 'maven'
 
-    version = '0.18.3-SNAPSHOT'
```

Comment:
did we even have a .4 or .3 stable?

Reply:
nope. What's the convention for stable releases? Do we increment from the last snapshot or no?